### PR TITLE
Server 3.2.1 release docs

### PR DESCRIPTION
**Pull request by rosieyohannan**
https://circleci.atlassian.net/browse/DOCSTEAM-44?atlOrigin=eyJpIjoiYzA2MTJmMTRiOTYzNGNiNTg2MzljMDlkMzY2NmE1YzMiLCJwIjoiaiJ9

### CODE REVIEW

File: jekyll/_cci2/server-3-whats-new.adoc [hunk 1]
```diff
@@ -19,6 +19,22 @@ patch releases and quarterly feature releases.
...
+== Release 3.2.1
+
+=== Upgrade notes
+
+From the KOTS admin console dashboard, select Version History from the menu bar and click **Deploy** for server v3.2.0.
+
+See <<Upgrade notes>> before upgrading from v3.1.x to v3.2.x.
+
+=== New features
+* Add support for private VMs on GCP
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-whats-new.adoc:31]:
```suggestion
* Private VMs are now supported for installations on GCP.
```

File: jekyll/_cci2/server-3-whats-new.adoc [hunk 2]
```diff
@@ -19,6 +19,22 @@ patch releases and quarterly feature releases.
...
+
+=== Fixes
+* mTLS is disabled by default
+* Increase SSH timeout for VMs to 10 minutes
+* Private VMs now request private IPs
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-whats-new.adoc:36]:
```suggestion
* mTLS is now disabled by default.
* SSH timeout for VMs has been increased to 10 minutes.
* Private VMs now request private IPs.
```

File: jekyll/_cci2/server-3-whats-new.adoc [hunk 3]
```diff
@@ -98,7 +114,6 @@ The new Traefik pos will then start ot schedule correctly.
 
 === Known issues
 
-* KOTS admin configuration incorrectly selects the Nomad mTLS as `enabled` during setup. It should be set to mTLS `disabled` until after nomad clients have been deployed.
```
Comment (ryanwohara) [inline jekyll/_cci2/server-3-whats-new.adoc:101]: This is still an issue for 3.2.0, just not 3.2.1

File: jekyll/_cci2/server-3-install-build-services.adoc [hunk 4]
```diff
@@ -520,7 +520,7 @@ This can be found using the following command:
 kubectl get service vm-service --namespace=<YOUR_CIRCLECI_NAMESPACE>
 ```
 ** *GCP project ID (required)* - 
-Name of the GCP project the cluster resides.
+Name of the GCP project the cluster resides. Below this you can uncheck the box if you want to use private VMs, which request private IPs.
```
Comment (thomasr8) [inline jekyll/_cci2/server-3-install-build-services.adoc:523]: Question....should this be "IP addresses"? Not sure which is why I am asking. There is another instance later in doc as well.

**Review by thomasr8 — APPROVED** [review]
Just had one comment about whether "Private IP" should be "Private IP address." Look good though 👍

File: jekyll/_cci2/server-3-whats-new.adoc (hunk 3 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server-3-whats-new.adoc:101]: Yeah this is a problem with the way I have this doc formatted, there's one known issues section for all releases which clearly doesn't work. I'll fix that.

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-whats-new.adoc:101]: Oh I'm talking nonsense, ignore me. I'll fix!

File: jekyll/_cci2/server-3-install-build-services.adoc (hunk 4 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-build-services.adoc:523]: you are quite right!

**Pull request merged**